components/multi-platform-controller/production/stone-prd-rh01/host-values.yaml

```diff
@@ -229,9 +229,58 @@ dynamicConfigs:
 
   linux-test-amd64:
     ami: ami-01aaf1c29c7e0f0af
+    instance-type: "m6idn.2xlarge"
+    sudo-commands: "/usr/bin/podman, /usr/bin/rm /usr/share/containers/mounts.conf"
```

Comment:
sudo-comands and user-data are missing in this machine type template, copy/paste them from any other root template.

Reply:
Done